diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
index 0000000000000000000000000000000000000000..b58b603fea78041071d125a30db58d79b3d49217
--- /dev/null
+++ b/.idea/.gitignore
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
diff --git a/.idea/workspace.xml b/.idea/workspace.xml
new file mode 100644
index 0000000000000000000000000000000000000000..94a25ca1fac6831258e9588d0f4977d044b38dc1
--- /dev/null
+++ b/.idea/workspace.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PropertiesComponent">{
+  &quot;keyToString&quot;: {
+    &quot;settings.editor.selected.configurable&quot;: &quot;settings.sync&quot;
+  }
+}</component>
+</project>
\ No newline at end of file
diff --git a/src/providers/appStore.js b/src/providers/appStore.js
index c3dc836fae10548a1d7122a835e0d60f3580a397..ac4c6b0eeb0ebc53c983fe502f378d7172028515 100644
--- a/src/providers/appStore.js
+++ b/src/providers/appStore.js
@@ -30,7 +30,7 @@ class AppStoreProvider implements IProvider {
       const dateNow = new Date().getTime();
 
       return fetch(
-        `https://itunes.apple.com/${countryCode}lookup?bundleId=${opt.packageName}&date=${dateNow}`,
+        `https://itunes.apple.com/${countryCode}/lookup?bundleId=${opt.packageName}&date=${dateNow}`,
         opt.fetchOptions
       )
         .then(res => res.json())
